editor/src/core/performance/performance-regression-tests.spec.tsx

```diff
@@ -127,7 +127,7 @@ describe('React Render Count Tests -', () => {
 
     const renderCountAfter = renderResult.getNumberOfRenders()
     // if this breaks, GREAT NEWS but update the test please :)
-    expect(renderCountAfter - renderCountBefore).toMatchInlineSnapshot(`989`)
+    expect(renderCountAfter - renderCountBefore).toMatchInlineSnapshot(`898`)
```

Comment:
interestingly, seems like on the CI these new snapshot values broke and it is indeed 989

Reply:
Yeah, that's weird, reran them and the snapshots updated back again.